aggregator/handler/handler.go

```diff
@@ -20,13 +20,47 @@
 
 package handler
 
+import (
+	"fmt"
+	"strings"
+)
+
 // Type is the handler type.
 type Type string
 
 // A list of supported handler types.
 const (
-	BlackholeHandler Type = "blackhole"
-	LoggingHandler   Type = "logging"
-	ForwardHandler   Type = "forward"
-	BroadcastHandler Type = "broadcast"
+	blackholeType Type = "blackhole"
```

Comment:
nit: any reason to not make these public? It seems a little odd that the configuration has a `Type` field, but none of these types are public although we unmarshal them correcly.

Reply:
Strictly speaking `Type` doesn't have to be public, I initially called it `handlerType` but considering it's inside the `handler` package, I got rid of the `handler` prefix.